src/QMCWaveFunctions/Fermion/SlaterDetBuilder.h

```diff
@@ -106,23 +106,47 @@ class SlaterDetBuilder : public WaveFunctionComponentBuilder
   template<typename VT,
            std::enable_if_t<(std::is_same<VT, ValueType>::value) &&
                             (std::is_floating_point<VT>::value), int> = 0>
-  void readCoeffs(hdf_archive& hin, std::vector<VT>& ci_coeff, size_t n_dets)
+  void readCoeffs(hdf_archive& hin, std::vector<VT>& ci_coeff, size_t n_dets,int ext_level)
   {
-    hin.read(ci_coeff, "Coeff");
+    std::string extVar;
+    extVar="Coeff_"+std::to_string(ext_level-1);
```

Comment:
Internal 0 is ground state while input 1 is the ground state introduce potential bugs.